internal/pkg/describe/env.go

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package describe
+
+import (
+	"encoding/json"
+	//"encoding/json"
+	"fmt"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+
+	//"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+)
+
+type Environment struct {
+	Name       string `json:"name"`
+	Production bool   `json:"production"` // add to instances used elsewhere (ie project_show)?
+	Region     string `json:"region"`
+	AccountID  string `json:"accountID"`
+}
+
+type EnvApp struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+}
+
+type EnvTag struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+type EnvSummary struct {
+	Environment  []*Environment `json:"environment"`
+	Applications []*EnvApp      `json:"applications"`
+	Tags         []*EnvTag      `json:"tags"`
+}
+
+// EnvDescriber retrieves information about an environment.
+type EnvDescriber struct {
+	env *archer.Environment
+
+	store           envGetter
+	ecsClient       map[string]ecsService
+	stackDescribers map[string]stackDescriber
+	sessProvider    sessionFromRoleProvider
+}
+
+// NewEnvDescriber instantiates an environment.
```

Comment:
Nit: comment should probably say "instantiates an environment describer", though I realize the comment in the webapp describer constructor is probably misleading.